tensornetwork/tensornetwork_test.py

```diff
@@ -16,874 +16,837 @@
 from __future__ import division
 from __future__ import print_function
 import tensornetwork
+import pytest
+import unittest
```

Comment:
Is `import unittest` needed? (haven't tried to run this yet)

Reply:
No it is not. Good catch.